app/lib/shared/versions.dart

```diff
@@ -21,6 +21,7 @@ final RegExp runtimeVersionPattern = RegExp(r'^\d{4}\.\d{2}\.\d{2}$');
 /// Make sure that at least two versions are kept here as the next candidates
 /// when the version switch happens.
 const acceptedRuntimeVersions = <String>[
+  // TODO: when removing, also remove SearchQuery.randomize
```

Comment:
What run runtimeVersion does that speak about... can we say when removing '2020.08.14'...